GDI spec 1.2 (#43)
signalfx/splunk-otel-lambda: RELEASING.md

> > The release process involves signing built packages and binaries and thus must be kept private and not exposed publicly.

Reviewer: I think the original wording could be misunderstood to mean that the packages and binaries must be kept private. How about this:

```suggestion
> Because the release process involves signing of output packages and binaries, the release process is kept private and not exposed publicly.
```

Author: @breedx-splk unfortunately this is copied 1:1 from the spec, so we should change the spec first :)

Reviewer: awesome. heh.
